codacy/values-production.yaml

```diff
@@ -1,14 +1,16 @@
 global:
   github: {}
-    # if you set this provider configuration, you must save it on the admin/integrations ui.
+    # If you set this provider configuration, you must save it on the admin/integrations UI.
     # enabled: "false"
+    # clientId: ""
+    # clientSecret: ""
```

Comment:
These configurations were missing in the `values-production.yaml` file, which is the one we are instructing the users to edit to configure their custom settings (https://codacy.github.io/chart/#2-installing-codacy).

Reply:
Maybe we can document for us all the values files that should be updated.

Reply:
Please see my comment here:

https://github.com/codacy/chart/pull/352/commits/ac8d36533e71696c704deaa05a91dd8d757025d7#r408760426

I agree this is currently not 100% clear, and I would like to have a definitive confirmation from @paulopontesm that what I wrote there is correct before documenting this better.